SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1012,6 +1012,12 @@ void USpatialReceiver::RemoveActor(Worker_EntityId EntityId)
 	{
 		// Force APlayerController::DestroyNetworkActorHandled to return false
 		PC->Player = nullptr;
+		
+		if (!NetDriver->IsServer())
+		{
+			GEngine->BroadcastNetworkFailure(NetDriver->GetWorld(), NetDriver, ENetworkFailure::ConnectionLost, 
+							 TEXT("PlayerController %s deleted. Server believes we have been timed out."), *PC->GetName());
```

Comment:
Couldn't this fire in the normal play dc flow - if the server deletes the pc before the client has dc-ed from spatial?

Reply:
Is that an issue?

Reply:
But wouldn't a real dc be caught by `USpatialReceiver::OnDisconnect` ? Which issues the same network failure broadcast.